modules/EmailMan/EmailMan.php

```diff
@@ -624,19 +645,21 @@ public function sendEmail($mail, $save_emails = 1, $testmode = false)
             $GLOBALS['log']->fatal('Encountered invalid email address: ' . $module->email1 . " Emailman id=$this->id");
             return true;
         }
-        
+
+
+        $SugarEmailAddress =$module->fromSugarEmailAddressField('email1');
         if (
                 $module instanceof Basic &&
                 $sugar_config['email_enable_confirm_opt_in'] &&
-                $module->fromSugarEmailAddressField('email1')->confirm_opt_in
+                $SugarEmailAddress->confirm_opt_in === ''
```

Comment:
It is intended. I moved some of the code outside in hopes to make it a little easier to understand. 

Since the requirements have changed to a drop down. The empty **''** means that they email address as not been opted in or confirm opted in. Which matches the body of the if statement.

Reply:
I have just realised that this is doing the same thing as the shouldBlockEmail method. So I think I will remove it ... and keep the next if statement, for backward compatibility reasons.